class-tgm-plugin-activation.php

```diff
@@ -1236,6 +1218,49 @@ public function notices() {
 		}
 
 		/**
+		 * Build a message string that specifies what actions are needed and the plugins that need those actions.
+		 *
+		 * @since 2.6.0
+		 *
+		 * @param  array  $message Associative array of actions: each key is an action and each value is an array of plugin slugs that need that action.
```

Comment:
Also, only a single space is needed after `@param`.

Yes, that means it won't line-up with the type in the `@return` string, but that's what WP does.